code/__DEFINES/mobfactions.dm

```diff
@@ -1,9 +1,90 @@
-/// If this seems rather...bare at the moment, its temporary.
-/// Later all mob factions will be moved here and their current factions replaced with faction defines.
+// Contains mob factions that are not handled by special role defines (for example, viscerators having ROLE_SYNDICATE)
 
+// Default factions
+
+/// Acts as a default faction for most violent creatures
+#define FACTION_HOSTILE "hostile"
+/// Acts as a default faction for most peaceful creatures
+#define FACTION_NEUTRAL "neutral"
+
+// Creature factions
+
+/// Ashwalker related creatures
+#define FACTION_ASHWALKER "ashwalker"
+/// Megafauna bosses of mining
+#define FACTION_BOSS "boss"
+/// CARPS
+#define FACTION_CARP "carp"
+/// Creatures summoned by chemical reactions
+#define FACTION_CHEMICAL_SUMMON "chemicalsummon"
```

Comment:
```suggestion
#define FACTION_CHEMICAL_SUMMON "chemical_summon"
```

i think snake_casing stuff makes the most sense here

Reply:
Alrighty, did the same for a few others too